src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java

```diff
@@ -372,7 +373,14 @@ public void validate(MonitorableJob.Status status) {
                 );
             } else {
                 FeedSource fs = Persistence.feedSources.getById(this.feedSourceId);
-                SharedStopsValidator ssv = new SharedStopsValidator(fs.retrieveProject());
+
+                // Get feed_id from feed version... Really awful hack!
+                JDBCFetcher feedFetcher = new JDBCFetcher("feed_info", null);
+                Object gtfsFeedId = feedFetcher.getResults(this.namespace, null, null).get(0).get("feed_id");
```

Comment:
I take it the feed_info will only ever have one row?

Reply:
I think if there are multiple we are ok just taking the first